packages/webservice/app/build.gradle.kts

```diff
@@ -18,7 +27,7 @@ repositories {
 dependencies {
     implementation(project(":models"))
     implementation("org.springframework.boot:spring-boot-starter-security")
-    implementation("com.yahoo.elide", "elide-spring-boot-starter", "5.0.0-pr18")
+    implementation("com.yahoo.elide", "elide-spring-boot-starter", "5.0.0-pr15")
```

Comment:
Why did you downgrade?  Also, latest version is 23.

Reply:
upgraded as per suggestion